types/context_test.go

```diff
@@ -44,18 +44,6 @@ func (l MockLogger) With(kvs ...interface{}) log.Logger {
 	panic("not implemented")
 }
 
-func TestContextGetOpShouldNeverPanic(t *testing.T) {
```

Comment:
Does this test need to be replicated?

Reply:
I deleted the Ops... no longer exist.
Also remove lots of enforced panics